### PR TITLE
55 add translation to length

### LINKED ISSUE #55: add translation to Length

### PR DESCRIPTION
**Pull request by bigotto**

### CODE REVIEW

File: src/messages/length/length.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { i18n } from '../../config/i18n';
+
+export default function Length(args: ValidationArguments) {
+  return i18n.__(
+    // eslint-disable-next-line no-nested-ternary
+    args.value.length < args.constraints[0]
+      ? 'LengthMin'
+      : args.value.length > args.constraints[1]
+      ? 'LengthMax'
+      : 'LenthBeetween',
```
Comment (bigotto) [inline src/messages/length/length.ts]: @vini-barba  if you found a better way to do this, we change! :)

File: jest.config.ts [hunk 2]
```diff
@@ -187,7 +187,7 @@ export default {
   // unmockedModulePathPatterns: undefined,
 
   // Indicates whether each individual test should be reported during the run
-  // verbose: undefined,
+  verbose: true,
```
Comment (vini-barba) [inline jest.config.ts]:
```suggestion
// verbose: undefined,
```

File: src/config/locales/pt.json [hunk 3]
```diff
@@ -9,5 +9,8 @@
   "IsDecimal": "{{property}} não é um valor decimal válido",
   "IsAscii": "{{property}} deve conter apenas caracteres ASCII",
   "IsNotEmpty": "{{property}} não deve ser vazio",
-  "IsEnum": "{{property}} deve ser um valor válido do enum"
+  "IsEnum": "{{property}} deve ser um valor válido do enum",
+  "LengthMin": "{{property}} deve ser maior ou igual a {{constraint}} caracteres",
+  "LengthMax": "{{property}} deve ser menor ou igual a {{constraint}} caracteres",
+  "LengthEach": "{{property}} deve ser  maior ou igual a {{constraint1}} e menor ou igual a {{constraint2}} caracteres"
```
Comment (vini-barba) [inline src/config/locales/pt.json]:
```suggestion
  "LengthEach": "{{property}} deve ser maior ou igual a {{constraint1}} e menor ou igual a {{constraint2}} caracteres"
```

**Review by vini-barba — CHANGES_REQUESTED** [review]

File: jest.config.ts (hunk 2 above)
Comment (bigotto) [reply jest.config.ts]: FIxed in last commit

**Review by vini-barba — APPROVED** [review]

**Pull request merged**